### PR TITLE
backport:  (scripted-diff) Merge bitcoin#21836, 21710, 21753

### PR DESCRIPTION
**Pull request by vijaydasmp**
bitcoin backports

### CODE REVIEW

File: src/immer/extra/guile/benchmark.scm [hunk 1]
```diff
@@ -54,7 +54,7 @@
          (display " seconds")
          (newline))))))
 
-(display ";;;; benchmarking creation...") (newline)
+(display ";;;; benchmarking creation…") (newline)
```
Comment (UdjinM6) [inline src/immer/extra/guile/benchmark.scm:57]: 21836: should exclude `src/immer/`

File: src/qt/dashstrings.cpp [hunk 2]
```diff
@@ -188,9 +188,9 @@ QT_TRANSLATE_NOOP("dash-core", "Failed to rescan the wallet during initializatio
 QT_TRANSLATE_NOOP("dash-core", "Failed to start a new mixing queue"),
 QT_TRANSLATE_NOOP("dash-core", "Failed to verify database"),
 QT_TRANSLATE_NOOP("dash-core", "Found enough users, signing ( waiting %s )"),
-QT_TRANSLATE_NOOP("dash-core", "Found enough users, signing ..."),
+QT_TRANSLATE_NOOP("dash-core", "Found enough users, signing …"),
```
Comment (UdjinM6) [inline src/qt/dashstrings.cpp:191]: 21836: should dashify script

File: src/init.cpp [hunk 3]
```diff
@@ -574,7 +574,7 @@ void SetupServerArgs(NodeContext& node)
     argsman.AddArg("-forcednsseed", strprintf("Always query for peer addresses via DNS lookup (default: %u)", DEFAULT_FORCEDNSSEED), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
     argsman.AddArg("-listen", "Accept connections from outside (default: 1 if no -proxy or -connect)", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
     argsman.AddArg("-listenonion", strprintf("Automatically create Tor onion service (default: %d)", DEFAULT_LISTEN_ONION), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
-    argsman.AddArg("-maxconnections=<n>", strprintf("Maintain at most <n> connections to peers (temporary service connections excluded) (default: %u)", DEFAULT_MAX_PEER_CONNECTIONS), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
+    argsman.AddArg("-maxconnections=<n>", strprintf("Maintain at most <n> connections to peers (temporary service connections excluded) (default: %u). This limit does not apply to connections manually added via -addnode or the addnode RPC, which have a separate limit of %u.", DEFAULT_MAX_PEER_CONNECTIONS, MAX_ADDNODE_CONNECTIONS), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
```
Comment (UdjinM6) [inline src/init.cpp:577]: 21710: missing changes in `src/rpc/net.cpp`

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
script in 21836 should be

```
-BEGIN VERIFY SCRIPT-
sed -i -E -e 's/\.\.\."\)(\.|,|\)| )/…"\)\1/' -- $(git ls-files -- 'src' ':(exclude)src/qt/dashstrings.cpp' ':(exclude)src/immer/')
sed -i -e 's/\.\.\.\\"/…\\"/' src/qt/sendcoinsdialog.cpp
sed -i -e 's|\.\.\.</string>|…</string>|' src/qt/forms/*.ui
sed -i -e 's|\.\.\.)</string>|…)</string>|' src/qt/forms/sendcoinsdialog.ui
-END VERIFY SCRIPT-
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
script fails (as expected) - you must drop all changes in `src/immer/*` and `src/qt/dashstrings.cpp` you introduced earlier

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK eeec358f9d5878cd276ee211914599e44f61415c

**Pull request merged**